packages/fiori/src/themes/SideNavigationItemBase.css

```diff
@@ -0,0 +1,254 @@
+.ui5-sn-item-ul {
+	margin: 0;
+	padding: 0;
+	list-style: none;
+}
+
+.ui5-sn-item {
+	display: flex;
+	align-items: center;
+	width: 100%;
+	box-sizing: border-box;
+	text-decoration: none;
+	position: relative;
+
+	height: var(--_ui5_side_navigation_item_height);
+	min-height: var(--_ui5_side_navigation_item_height);
+	cursor: pointer;
+	color: inherit;
+	background-color: var(--sapList_Background);
+	border-radius: var(--_ui5_side_navigation_item_border_radius);
+	transition: var(--_ui5_side_navigation_item_transition);
+
+	margin-block-end: var(--_ui5_side_navigation_item_bottom_margin);
+}
+
+:host(.ui5-sn-item-hidden[side-nav-collapsed]),
+:host([side-nav-collapsed]) .ui5-sn-item-hidden {
+	display: none;
+}
+
+.ui5-sn-item:focus {
+	outline: none;
+}
+
+.ui5-sn-item:focus::after {
+	border: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
+	position: absolute;
+	content: "";
+	inset: var(--_ui5_side_navigation_item_focus_border_offset);
+	z-index: 2;
+	pointer-events: none;
+	border-radius: var(--_ui5_side_navigation_item_focus_border_radius);
+}
+
+.ui5-sn-item:not(.ui5-sn-item-disabled).ui5-sn-item-selected:focus::before {
+	border: var(--_ui5_side_navigation_selected_and_focused_border_style_color);
+}
+
+.ui5-sn-item.ui5-sn-item-group::before,
+.ui5-sn-item.ui5-sn-item-level1::before {
+	border: var(--_ui5_side_navigation_group_border_style_color);
+	border-width: var(--_ui5_side_navigation_group_border_width);
+}
+
+.ui5-sn-item-group[aria-expanded="true"]::before,
+.ui5-sn-item-level1[aria-expanded="true"]::before {
+	border-width: var(--_ui5_side_navigation_group_expanded_border_width);
+}
+
+/* borders are drawn using a pseudo element.
+in some themes, the border is drawn only on the bottom.
+in high contrast themes the border is on all sides and there is extra border on hover */
```

Comment:
I suggest:
In high contrast themes, the border is present on all sides, and there is an additional border that appears on hover.